includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -434,43 +474,86 @@ protected function setup_assets() {
 			new Script(
 				'googlesitekit-modules-reader-revenue-manager',
 				array(
-					'src'          => $base_url . 'js/googlesitekit-modules-reader-revenue-manager.js',
-					'dependencies' => array(
+					'src'           => $base_url . 'js/googlesitekit-modules-reader-revenue-manager.js',
+					'dependencies'  => array(
 						'googlesitekit-vendor',
 						'googlesitekit-api',
 						'googlesitekit-data',
 						'googlesitekit-modules',
+						'googlesitekit-notifications',
 						'googlesitekit-datastore-site',
 						'googlesitekit-datastore-user',
 						'googlesitekit-components',
 					),
+					'load_contexts' => array( Asset::CONTEXT_ADMIN_SITEKIT, Asset::CONTEXT_ADMIN_POST_EDITOR ),
 				)
 			),
 		);
 
-		if ( Feature_Flags::enabled( 'rrmModuleV2' ) ) {
+		if ( Feature_Flags::enabled( 'rrmModuleV2' ) && Block_Support::has_block_support() ) {
 			$assets[] = new Script(
-				'googlesitekit-reader-revenue-manager-block-editor-plugin',
+				'blocks-reader-revenue-manager-block-editor-plugin',
 				array(
 					'src'           => $base_url . 'js/blocks/reader-revenue-manager/block-editor-plugin/index.js',
 					'dependencies'  => array(
 						'googlesitekit-data',
 						'googlesitekit-i18n',
 						'googlesitekit-modules',
+						'googlesitekit-modules-reader-revenue-manager',
 					),
 					'execution'     => 'defer',
 					'load_contexts' => array( Asset::CONTEXT_ADMIN_POST_EDITOR ),
 				)
 			);
 
 			$assets[] = new Stylesheet(
-				'googlesitekit-block-editor-css',
+				'blocks-reader-revenue-manager-block-editor-plugin-styles',
 				array(
 					'src'           => $base_url . 'js/blocks/reader-revenue-manager/block-editor-plugin/editor-styles.css',
 					'dependencies'  => array(),
 					'load_contexts' => array( Asset::CONTEXT_ADMIN_POST_EDITOR ),
 				)
 			);
+
+			$assets[] = new Script(
+				'blocks-contribute-with-google',
+				array(
+					'src'           => $base_url . 'js/blocks/reader-revenue-manager/contribute-with-google/index.js',
+					'dependencies'  => array(
+						'googlesitekit-data',
+						'googlesitekit-i18n',
+						'googlesitekit-modules',
+						'googlesitekit-modules-reader-revenue-manager',
+					),
+					'load_contexts' => array( Asset::CONTEXT_ADMIN_POST_EDITOR ),
+					'execution'     => 'defer',
+				)
+			);
+
+			$assets[] = new Script(
+				'blocks-subscribe-with-google',
+				array(
+					'src'           => $base_url . 'js/blocks/reader-revenue-manager/subscribe-with-google/index.js',
+					'dependencies'  => array(
+						'googlesitekit-data',
+						'googlesitekit-i18n',
+						'googlesitekit-modules',
+						'googlesitekit-modules-reader-revenue-manager',
+					),
+					'load_contexts' => array( Asset::CONTEXT_ADMIN_POST_EDITOR ),
+					'execution'     => 'defer',
+				)
+			);
+
+			// TODO: Find a way to enqueue this stylesheet in the site editor.
```

Comment:
There are a number of `TODO` comments in this PR, mostly related to the site editor support, including this one. Just to confirm, are we okay to keep them for now, provided that site editor support will be added via a new issue?

Reply:
Hmm good question; I've removed the comments related to the site editor, but I think the other two TODO comments are useful to keep for now, although I have made a note to create an issue to address the point they relate to (we should style the RRM frontend button wrapper using a stylesheet rather than inline styles).